worker/singular/manifold.go

```diff
@@ -22,10 +22,11 @@ type ManifoldConfig struct {
 	Clock         clock.Clock
 	APICallerName string
 	Duration      time.Duration
-	Claimant      names.MachineTag
-	Entity        names.Tag
+	// TODO - claimaint should be a ControllerAgentTag
```

Comment:
change to `Claimant ControllerAgentTag`?

Reply:
We can't because we still have MachineTags for IAAS